### ISSUE TITLE
input count table

### ISSUE THREAD

**Issue by raqxavier** [kevinmcgregor/mdine#15]
Good morning. The input count data should be not "normalized" right?

Thanks in advance
Raquel

**Comment by kevinmcgregor** [comment]
Hi Raquel,

Correct, it should be the raw taxa/OTU table input to the mdine function.

Kevin

**Comment by raqxavier** [comment]
Thank you for your response. Just to be sure I am interpreting my results correctly, if adj0 and adj1 matrices contain only 0's, this means that the structure of the matrices was not significantly affected by the binary variable, correct?  

Thanks in advance
Raquel

**Comment by kevinmcgregor** [comment]
That means that the matrix for each individual group was estimated to have no connections.  This would usually be due to a smaller sample size and therefore an underpowered analysis.

**Comment by raqxavier** [comment]
Yes I have limited representation of each group in this dataset. Thank you for your reply.
